common/src/node.rs

```diff
@@ -77,6 +96,39 @@ impl Graph {
         Some((chunk, coords))
     }
 
+    /// Populates a chunk with the given voxel data and ensures that margins are correctly removed
+    /// if necessary.
+    pub fn populate_chunk(&mut self, chunk: ChunkId, mut new_data: VoxelData, modified: bool) {
```

Comment:
`modified` never seems to be set to `true`. Does its introduction belong in a later commit?

Reply:
`modified` is set to true at https://github.com/Ralith/hypermine/pull/317/files/410344b5bc303361e0854a05b78ea85f4e3ad1af..fe227b1c1149edb2c6aff48248a54d4683e4a736#diff-ec354b5124aea7e933d8246564d903169084ba2fc08c7e2801d066854b5ba827R154.

Reply:
I'm referring to the argument to this function. It only ever seems to be passed a literal `false` until the next commit. No need to shuffle things around; it's just easier to follow the intention behind a change if it's used.

Reply:
Ah, I missed that. Thanks for clarifying.

Reply:
I ended up shuffling things around anyway, as I like trying to make the commit history as readable as possible.